### PR TITLE
Add Kommander 1.1.0 RC 4 Release Notes

### PR DESCRIPTION
**Pull request by juliangieseke**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 1]
```diff
@@ -50,6 +50,49 @@ Template:
...
+### Kommander 1.1.0 RC 4 - June 26th 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.15.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+#### Features/Improvements
+* Add guideance what to do when cluster deletion failed
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:62]:
```suggestion
* Add guidance on what to do when cluster deletion fails
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 2]
```diff
@@ -50,6 +50,49 @@ Template:
...
+* Allow default federated addon values to be overriden
+* Attach clusters to Kommander is now considered GA
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:64]:
```suggestion
* Attaching clusters to Kommander is now considered GA
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 3]
```diff
@@ -50,6 +50,49 @@ Template:
...
+* Update kubecost web urls
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:65]:
```suggestion
* Update Kubecost web urls
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 4]
```diff
@@ -50,6 +50,49 @@ Template:
...
+* Add flag (`--skip-credentials-display`) to Kommander to Konvoy it displaying login information in Logs
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:66]:
```suggestion
* Add flag (`--skip-credentials-display`) to Kommander so Konvoy does not display login information in logs
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 5]
```diff
@@ -50,6 +50,49 @@ Template:
...
+* Improved context handling for multiple contexts when attaching cluster
+
+#### Bug Fixes
+* Kommander Grafana was unavailable after self-attaching host cluster as managed cluster
+* Fixed a bug where Kommander addon wasnt successfully deploying on Azure
+* Fix Kommander to not show unofficially supported versions of Kubernetes by default
+* Fix possible data collision bug related to clusters
+* Fix a possible crash-loop situation for kubeaddons when cert-manager isnt ready yet
+* Fix and Improve LDAP Identity provider handling
+* Fix display of "nothing to report" situations vs acutal errors
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:78]:
```suggestion
* Fix display of "nothing to report" situations vs actual errors
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 6]
```diff
@@ -50,6 +50,49 @@ Template:
...
+* Allow default federated addon values to be overriden
+* Attach clusters to Kommander is now considered GA
+* Update kubecost web urls
+* Add flag (`--skip-credentials-display`) to Kommander to Konvoy it displaying login information in Logs
+* Add ability to disable federation of monitoring stack
+* Added autogenerated labels to advanced cluster create form
+* Improved context handling for multiple contexts when attaching cluster
+
+#### Bug Fixes
+* Kommander Grafana was unavailable after self-attaching host cluster as managed cluster
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:72]: Is this the detach Kommander bug? Was it only applicable to grafana? I thought all the ingresses stopped working?

Comment (juliangieseke) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:72]: not 100% sure  tbh, thats what the JIRA was talking about. I know we fixed UI a while ago (probbaly before 1.0.0), not sure if grafana was missed or UI was the only one we fixed tho

**Comment by juliangieseke** [comment]
yup, but lets get this one in first

**Pull request merged**